index.html

```diff
@@ -75,25 +84,38 @@ <h1 class="text-primary">Chryz-Hub</h1>
       <div class="container">
         <div class="vision-content">
           <h2 class="text-primary">Our Vision</h2>
-          <p class="text-secondary">Our vision is to add value, solve problems, render help and support to developers, open source contributors and everyone at large and also raise and build Open Source Giants.</p>
-          <a href="https://github.com/chryz-hub/chryz-hub.github.io/blob/master/vision.md" class="btn btn-primary btn-rounded" re="noopener" target="_blank">Read more</a>  
+          <p class="text-secondary">Our vision is to add value, solve problems, render help and support to developers,
+            open source contributors and everyone at large and also raise and build Open Source Giants.</p>
+          <a href="https://github.com/chryz-hub/chryz-hub.github.io/blob/master/vision.md"
+            class="btn btn-primary btn-rounded" re="noopener" target="_blank">Read more</a>
         </div>
       </div>
     </section>
     <!-- Buttons  -->
     <div class="container buttons">
-      <a href="https://github.com/chryz-hub/chryz-hub.github.io/issues/new?assignees=&labels=invite+me+to+the+organization&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization" rel="noopener" class=" btn btn-primary">Join the Community</a>
-      <a href="https://github.com/chryz-hub/opensource-4-everyone/blob/master/My-PR-Contribution-Practice.md" rel="noopener" class="btn btn-secondary" target="_blank"><i class="fas fa-code-branch"></i> Create Pull Request</a>
+      <a href="https://github.com/chryz-hub/chryz-hub.github.io/issues/new?assignees=&labels=invite+me+to+the+organization&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization"
+        rel="noopener" class=" btn btn-primary">Join the Community</a>
+      <a href="https://github.com/chryz-hub/opensource-4-everyone/blob/master/My-PR-Contribution-Practice.md"
+        rel="noopener" class="btn btn-secondary" target="_blank"><i class="fas fa-code-branch"></i> Create Pull
+        Request</a>
     </div>
   </header>
   <main>
+
     <section id="projects" class="projects">
       <h2>Projects</h2>
-      <div class="flex-items">
+
+      <div class="flex-col-2" id="githubprojects">
+        <!-- GitHub Projects from scripts.js goes here -->
+      </div>
+
+      <!-- Not needed if we go to api call in the js -->
```

Comment:
Project cards are not needed in the index html due we edit the html dom direktly in the scrpit

Reply:
Right, we just created them for reference. We can remove them once they are added through the script.

Reply:
should we go with this project card design?

Reply:
That's fine. But, adding bootstrap now would add unnecessary styling. Wouldn't it?